packages/titanium-docgen/index.js

```diff
@@ -1020,18 +1018,28 @@ formats.forEach(function (format) {
 			if (cssFile) {
 				fs.createReadStream(cssPath).pipe(fs.createWriteStream(pathMod.join(output, cssFile)));
 			}
-			const imgPath = pathMod.join(__dirname, 'images');
-			if (os.type() === 'Windows_NT') {
-				copyCommand = `xcopy ${imgPath} ${output}`;
-				copyCommand = copyCommand.replace(/\//g, '\\') + ' /s';
-			} else {
-				copyCommand = `cp -r ${imgPath} ${output}`;
-			}
 
-			exec(copyCommand, function (error) {
-				if (error !== null) {
-					common.log(common.LOG_ERROR, 'Error copying file: %s', error);
+			// For each of the input directories, check if there's an images subdir
+			// if so, copy it over!
+			basePaths.concat(addOnDocs).forEach(p => {
+				const imgPath = pathMod.join(p, 'images');
+				if (!fs.existsSync(imgPath)) {
+					return;
+				}
+				// TODO: Use fs module to copy it!
```

Comment:
Is there anything that prevents us from simply using fs-extra's `copy` for example? Should make this a simple one-liner.

Reply:
I wanted to simply do the minimum here. But I totally agree - this code needs to be refactored badly. I'd suggest we try and run ESLint, use async/await, move to fs-extra, etc. I can do that in a follow-on PR. Though I'm a little hesitant given we have no tests...

Reply:
Yeah, that's true. Let's tackle this at a later time.